airflow/dags/polygonetl_airflow/build_export_dag.py

```diff
@@ -29,6 +29,7 @@ def build_export_dag(
         provider_uris_archival,
         output_bucket,
         export_start_date,
+        custom_dag = DAG,
```

Comment:
An idea: maybe something like `dag_factory` would be a more appropriate name for this variable? After all, by default you're providing the value of `DAG` there which is not custom.

Reply:
what about `default_dag` the word `factory` in `dag_factory` could be a bit misleading @pswies  ?

Reply:
Hm. `default` is the opposite of `custom` and you want to enable customomization... `dag_class`?